client/templates/study_groups/all_study_groups.html

```diff
@@ -7,7 +7,7 @@
           <div class="row">
               <div class="col-md-12 align-center">
                 <h2>Study Groups</h2>
-                <div class="btn btn-cb2" id="createGroupButton"> {{_ "start_new_group"}} </div>
+                <div class="btn btn-cb2 {{#unless currentUser}} continue-popup {{/unless}}" id="createGroupButton"> {{_ "start_new_group"}} </div>
```

Comment:
Please add this `id="{{#if currentUser}}createGroupButton{{/if}}"`. and we won't be needing a 
`if (Meteor.userId()) {` block in our helper function.
Thanks.